barretenberg/cpp/bootstrap.sh

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env bash
 source $(git rev-parse --show-toplevel)/ci3/source_bootstrap
 
+# I (benesjan) temporarily disabled caching to avoid issues with stale AVM transpiler.
+export NO_CACHE=1
```

Comment:
Chatted with @ludamad and the issue with the AVM transpiler is caused by broken caching in BB. This is a temporary workaround to see if CI passes.

Reply:
fixed in https://github.com/AztecProtocol/aztec-packages/pull/18223